Temporarily disabled checkstyle for release

@bjoernhaeuser I realize this is bad practice but I temporarily disabled checkstyle while performing a release.  I got the following errors for an "injected test".

```
    [ERROR] target/generated-test-sources/injected/InjectedTest.java:[1] (imports) AvoidStarImport: Using the '.*' form of import should be avoided - java.util.*.
    [ERROR] target/generated-test-sources/injected/InjectedTest.java:[9,44] (whitespace) WhitespaceAround: '+' is not preceded with whitespace.
    [ERROR] target/generated-test-sources/injected/InjectedTest.java:[9,45] (whitespace) WhitespaceAround: '+' is not followed by whitespace.
    [ERROR] target/generated-test-sources/injected/InjectedTest.java:[11,30] (whitespace) WhitespaceAfter: ',' is not followed by whitespace.
    [ERROR] target/generated-test-sources/injected/InjectedTest.java:[12,33] (whitespace) WhitespaceAfter: ',' is not followed by whitespace.
    [ERROR] target/generated-test-sources/injected/InjectedTest.java:[13,32] (whitespace) WhitespaceAfter: ',' is not followed by whitespace.
    [ERROR] target/generated-test-sources/injected/InjectedTest.java:[14,38] (whitespace) WhitespaceAfter: ',' is not followed by whitespace.
    [ERROR] target/generated-test-sources/injected/InjectedTest.java:[15,42] (whitespace) WhitespaceAfter: ',' is not followed by whitespace.
    [ERROR] target/generated-test-sources/injected/InjectedTest.java:[16,32] (whitespace) WhitespaceAfter: ',' is not followed by whitespace.
    [INFO] ------------------------------------------------------------------------
    [INFO] BUILD FAILURE
    [INFO] ------------------------------------------------------------------------
    [INFO] Total time: 19.641 s
    [INFO] Finished at: 2018-01-21T22:17:27-08:00
    [INFO] Final Memory: 264M/1792M
    [INFO] ------------------------------------------------------------------------
    [ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check (validate) on project ghprb: You have 9 Checkstyle violations. -> [Help 1]
    [ERROR] 
    [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
    [ERROR] Re-run Maven using the -X switch to enable full debug logging.
    [ERROR] 
    [ERROR] For more information about the errors and possible solutions, please read the following articles:
    [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

While running the following command:

```
mvn org.apache.maven.plugins:maven-release-plugin:2.5:prepare org.apache.maven.plugins:maven-release-plugin:2.5:perform
```

I added `-Dcheckstyle.skip` for now because I'm not sure how to handle generated classes like that.

--- a/pom.xml
+++ b/pom.xml
@@ -34,13 +34,13 @@
             <timezone>America/Los_Angeles</timezone>
         </developer>
     </developers>
+
     <scm>
         <connection>scm:git:ssh://github.com/jenkinsci/ghprb-plugin.git</connection>
         <developerConnection>scm:git:ssh://git@github.com/jenkinsci/ghprb-plugin.git</developerConnection>
         <url>https://github.com/jenkinsci/ghprb-plugin</url>
         <tag>HEAD</tag>
     </scm>
-
 
     <issueManagement>
         <system>GitHub</system>
@@ -57,6 +57,18 @@
         <checkstyle.version>6.7</checkstyle.version>
         <checkstyle.plugin.version>2.17</checkstyle.plugin.version>
     </properties>
+
+    <!--
+        Activate profiles which override properties.
+    -->
+    <profiles>
+        <profile>
+            <id>release</id>
+            <properties>
+                <checkstyle.skip>true</checkstyle.skip>
+            </properties>
+        </profile>
+    </profiles>
 
     <dependencies>
 
@@ -219,6 +231,10 @@
             <plugin>
                 <artifactId>maven-release-plugin</artifactId>
                 <version>2.5</version>
+                <configuration>
+                    <!-- During release:perform, enable the "release" profile -->
+                    <releaseProfiles>release</releaseProfiles>
+                </configuration>
                 <dependencies>
                     <dependency>
                         <groupId>org.apache.maven.scm</groupId>

